packages/svg-sprites/unfill.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Can we have a unified treatment for icons, at least within svg type? If I recall the `none` fill comes from the root `svg` element, which is beneficial to remove for size in all cases. The `212121` fill comes from the `path`. In the original package we opted to specify no fill to remove any _opinion_ on color and allow consumers to (likely) set to `currentcolor` in their style sheets. I suggest going forward with that since it would work in this new package, and would also make the download size smaller. I do agree that `currentcolor` is probably very common, but since it's not a given I would lean toward being more concise.

Can we share the file from the other package or move it to a utilities directory?

Reply:
I'll look into that, thanks for the feedback.